doc/classes/Environment.xml

```diff
@@ -9,7 +9,11 @@
 		- Glow
 		- Tonemap (Auto Exposure)
 		- Adjustments
-		These effects will only apply when the [Viewport]'s intended usage is "3D" or "3D Without Effects". This can be configured for the root Viewport with [member ProjectSettings.rendering/quality/intended_usage/framebuffer_allocation], or for specific Viewports via the [member Viewport.usage] property.
+		With "2D Without Sampling", all post-processing effects will be turned off. With "3D Without Effects", these options will be invalid:
+		- Ssao
+		- Ss reflections
+		This can be done using [member ProjectSettings.rendering/quality/intended_usage/framebuffer_allocation] or via the [member Viewport.usage] property to configure the root viewport for a specific viewport.
```

Comment:
```suggestion
		This can be configured for the root Viewport with [member ProjectSettings.rendering/quality/intended_usage/framebuffer_allocation], or for specific Viewports via the [member Viewport.usage] property.
```

I would revert to the previous language for this sentence. The ProjectSetting sets the allocation mode for the root viewport while the Viewport.usage setting can be used to set it for a viewport that is not the root viewport. The two settings apply in different places so it is incorrect to say that you can use either.